api/src/main/java/org/openmrs/module/patientflags/api/impl/FlagServiceImpl.java

```diff
@@ -657,14 +658,9 @@ public void deletePatientFlagsForFlag(Flag flag) throws DAOException {
 
 	@Override
 	public List<Flag> getFlagsForPatient(Patient patient, Filter filter) {
-		List<Flag> filteredFlags = new ArrayList<Flag>();
-		
-		List<Flag> flags = getFlagsForPatient(patient);
-		for (Flag flag : filter.filter(flags)) {
-			filteredFlags.add(flag);
-		}
-		
-		return filteredFlags;
+		return 	getFlagsForPatient(patient).stream()

```

Comment:
Can't we just use `return filter.filter(getFlagsForPatient(patient));` here?

Reply:
yes, we can. I update the code with this changes